KnowSheet/scripts/Makefile

```diff
@@ -26,7 +27,7 @@ BIN=$(SRC:%.cc=.noshit/%)
 
 OS=$(shell uname)
 ifeq ($(OS),Darwin)
-  CPPFLAGS+= -x objective-c++ -fobjc-arc
+  CPPFLAGS+= -stdlib=libc++ -x objective-c++ -fobjc-arc
```

Comment:
I used XCode's clang and it's ok. But I didn't try to compile it with macport's clang